src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/AddDisableRuntimeMarshallingAttributeFixerTests.cs

```diff
@@ -102,21 +102,25 @@ struct Native
         private static async Task VerifyCodeFixAsync(string source, string? propertiesFile, string? expectedPropertiesFile, DiagnosticResult diagnostic)
         {
             var test = new Test();
+
+            string normalizedSource = LineEndingsHelper.Normalize(source);
```

Comment:
LGTM - I'll let interop owners review this - my only thought is that maybe the Normalize call could be inside Test(), so the tests don't all need it.

Reply:
@danmoseley  I checked if I could put it inside Test but it seems like Test is on a different project (`Microsoft.CodeAnalysis.Testing`) and I didn't know if this would break/modify code somewhere else, and I certainly didn't want to mess something up.

Reply:
@jkoritzinsky or @elinor-fung Do either of you have thoughts on where we could normalize the test source for this scenario? Ideally it would be in a single location we could easily comment.

Reply:
The `LibraryImportGenerator.UnitTests` use a subclass of the class in `Microsoft.CodeAnalysis.Testing`:
https://github.com/dotnet/runtime/blob/c6a51bd50163e14d4d8fcd7ab133257724b24fff/src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/Verifiers/CSharpCodeFixVerifier.cs#L97

I think that would be a reasonable place. It could define its own `RunAsync`. It looks like the tests always use `CancellationToken.None` right now, so we probably don't even need that parameter and can rely on the default value of the base class' `RunAsync`. Something like:

```csharp
public async Task RunAsync()
{
    // Normalize line endings
    TestCode = LineEndingsHelper.Normalize...
    FixedCode = LineEndingsHelper.Normalize... 
    BatchFixedCode = LineEndingsHelper.Normalize...

    await base.RunAsync();
}
```